content/en/tracing/setup_overview/custom_instrumentation/dotnet.md

```diff
@@ -24,39 +24,38 @@ further_reading:
       text: '.NET code samples'
 ---
 <div class="alert alert-info">
-For instructions on how to setup the .NET Tracer and enable automatic instrumentation, see the <a href="https://docs.datadoghq.com/tracing/setup/dotnet/">.NET setup instructions</a>.
+For instructions on how to setup the .NET Tracer and enable automatic instrumentation, see the <a href="https://docs.datadoghq.com/tracing/setup/dotnet-framework/">.NET Framework setup instructions</a> or the <a href="https://docs.datadoghq.com/tracing/setup/dotnet-core/">.NET Core setup instructions</a>.
 </div>
 
-**Note:** When using both custom and automatic instrumentation, it is important to keep the MSI installer and NuGet package versions in sync.
+This page details common use cases for adding and customizing observability with Datadog APM. Custom instrumentation is supported on **.NET Framework 4.6.1+** for Windows, on **.NET Core 2.0+** for Windows and Linux, and on **.NET 5** and **.NET 6** for Windows and Linux. To perform the steps outlined below, you may need to add NuGet package references to one or more of the following libraries:
```

Comment:
I don't think we should list compatibility here, but only refer people to the compatibility requirement page.
That will allow us not to forget this when we udapte compatibility requirements.